### ISSUE TITLE
Accessibility (Medium) - Programmatically Determined Structure (Guideline 1.3.1)

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#142]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/142) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

Please comb the entire interface and add roles, icons, and headings. See below for further explanation. 

**Headings:** Heading tags should be in chronological order. For example, I noticed that on the “Manage Administrators” page that <h3>Labor Administrators</h3> follows an <h4>Add or remove administrators here</h4>. The <h4> needs to be removed and replaced with the same effect in CSS. _This is the only place I noticed, but please double-check my work._

**Roles for Regions:** This article explains roles better than I ever could: [https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11). Please ensure that everything that needs roles has them. One example that I know is needed is the search bars. They need to be labeled role=“search”.

**Icons:** Please implement this \([https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24)\) with the icons.

If you need more information about this guideline, please visit the wiki \([http://172.31.2.178/accessibility/checklist](http://172.31.2.178/accessibility/checklist)\) or ask Hailey.

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

The first two parts of the issue are resolved in this PR: [https://bitbucket.org/laborstudents/lsf-flask/pull-requests/133](https://bitbucket.org/laborstudents/lsf-flask/pull-requests/133)  
The last part is being solved on branch _issue142icons_

‌

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

PR is in for the last part of the issue. Waiting to be merged.

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

PR is merged: https://bitbucket.org/laborstudents/lsf-flask/pull-requests/147
